Routing/Loader/RestXmlCollectionLoader.php

```diff
@@ -207,9 +207,10 @@ public function supports($resource, $type = null)
      */
     protected function validate(\DOMDocument $dom)
     {
-        $restRoutinglocation = realpath(__DIR__.'/../../Resources/config/schema/routing/rest_routing-1.0.xsd');
+        $configBasePath = realpath($this->locator->locate('../../../Resources/config'));
```

Comment:
That would be perfect but will not work out of a Symfony project, and so break the test suit, right ?